corehq/util/context_processors.py

```diff
@@ -87,7 +87,7 @@ def websockets_override(request):
         return {}
 
 
-def enterprise_mode(request):
+def django_settings(request):
     return {
-        'enterprise_mode': settings.ENTERPRISE_MODE
+        'django_settings': settings
```

Comment:
It just makes it easier to expose secrets to the world. A typo that got missed in code review could put our entire settings dict into a template. Best to be explicit about what templates should have access to.